controller/controller.go

```diff
@@ -840,6 +841,9 @@ func main() {
 	// init rest server context before listening KV object store, as federation server can be started from there.
 	rest.InitContext(&rctx)
 
+	// Assign callback so cert manager can receive cert changes.
+	cctx.NotifyCertChange = rest.CertManager.NotifyChanges
```

Comment:
Is there a specific reason to assign the callback here, and not in the init part?